apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -179,10 +213,9 @@ const REDUX_TRACKING = {
 	'core/block-editor': {
 		moveBlocksUp: getBlocksTracker( 'wpcom_block_moved_up' ),
 		moveBlocksDown: getBlocksTracker( 'wpcom_block_moved_down' ),
-		removeBlocks: getBlocksTracker( 'wpcom_block_deleted' ),
-		removeBlock: getBlocksTracker( 'wpcom_block_deleted' ),
+		removeBlocks: trackBlockRemoval,
+		removeBlock: trackBlockRemoval,
 		moveBlockToPosition: getBlocksTracker( 'wpcom_block_moved_via_dragging' ),
-		deleteBlock: getBlocksTracker( 'wpcom_block_deleted' ),
```

Comment:
`deleteBlock` is nowhere to be found in the Gutenberg codebase. Might have been added as an accident.